UPGRADE-5.2.md

```diff
@@ -75,6 +75,7 @@ Notifier
  * [BC BREAK] Changed the type-hint of the `$recipient` argument in `NotifierInterface::send()`,
    `Notifier::getChannels()`, `ChannelInterface::notifiy()` and `ChannelInterface::supports()` to
    `RecipientInterface`.
+ * symfony/slack-notifier: Deprecate `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```

Comment:
I would also add an UPGRADE file to the bridge itself, because it is a separate package.

WDYT @derrabus ?

Reply:
I think this is a good idea.

Reply:
Symfony only has seperate CHANGELOG files. The UPGRADE file is shared for all subtree splits in the root

Reply:
Ok is the prefix with the package name fine? Or should it be a subheading?

Reply:
> Apart from that, the deprecation was added in 5.1, so the entry should be moved to UPGRADE-5.1.md.

🙈  You are right, I mixed it up.